app/healthcheck.py

```diff
@@ -4,7 +4,6 @@
 healthcheck_blueprint = Blueprint('healthcheck', __name__, url_prefix='')
 
 
-@healthcheck_blueprint.route('/')
```

Comment:
No Kubernetes health check monitoring actually check the `/` root for check check: we check for `_status`.

Reply:
Possibly they may have meant this to be a pseudo especially for redirects..

You sure about this??

Reply:
The redirect was probably presented as a convenience. For security purpose though, we want to strip most endpoints so that vulnerability scanners hit a wall.